src/main/java/org/web3j/tx/ManagedTransaction.java

```diff
@@ -75,8 +81,13 @@ protected TransactionReceipt processResponse(EthSendTransaction transactionRespo
         }
 
         String transactionHash = transactionResponse.getTransactionHash();
-
-        return waitForTransactionReceipt(transactionHash);
+        if (attempts > 0) {
```

Comment:
In Ethereum I reached 60+ transactions per minute for some simple smart contract, with Quorum I executed 100-150 transactions per second.  Before I tried with web3j, and I disabled the polling for each transaction because I never reached more than 25 transactions per minute, my laptop was frozen because the polling made all network connections busy and the system refused to accept new connections until I closed nodejs.